tests/functional/workloads/cnv/test_multi_vm_configurations.py

```
@@ -84,42 +93,44 @@ def test_cnv_vms(self, setup_cnv, setup):
         """
         all_vm_list = self.vm_objs_def + self.vm_objs_aggr
 
-        # 1.Validate data integrity using md5sum.
+        logger.test_step("Validate data integrity using md5sum")
         file_name = "/tmp/dd_file"
```

Ruff (0.15.21)

[error] Probable insecure usage of temporary file or directory: "/tmp/dd_file"

(S108)